packages/flutter_tools/lib/src/windows/visual_studio.dart

```diff
@@ -23,11 +23,16 @@ class VisualStudio {
     required Logger logger,
   }) : _platform = platform,
        _fileSystem = fileSystem,
-       _processUtils = ProcessUtils(processManager: processManager, logger: logger);
+       _processUtils = ProcessUtils(processManager: processManager, logger: logger),
+       _logger = logger;
 
   final FileSystem _fileSystem;
   final Platform _platform;
   final ProcessUtils _processUtils;
+  final Logger _logger;
+
+  /// Matches the description property from the vswhere.exe JSON output.
+  final RegExp _vswhereDescriptionProperty = RegExp(r'\s*"description":\s*".*"\s*,?');
```

Comment:
Per JSON spec (if not current vswhere.exe behavior) there could be whitespace between the key and colon of an object

Reply:
Good catch!

Reply:
Fixed